account_invoice_show_currency_rate/models/account_move.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    currency_rate_amount = fields.Float(
+        string="Rate amount",
+        compute="_compute_currency_rate_amount",
+        store=True,
```

Comment:
Don't do it `store`, or all existing invoices will need to recompute its value on installation/migration. Any reason for doing it stored that I lose?

Reply:
Ok, i have removed `store=True` because is not really necessary.